prod-stack-configuration.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Parameters" : {
+    "EnvType" : "production"
+  },
+  "Tags" : {
+    "Name" : "testlibnd-appstream-gateway",
+    "Contact" : "lib-esu-group@nd.edu",
+    "Owner" : "wsill",
```

Comment:
I support you in blaming Bill for everything that might happen with this pipeline 💯

Reply:
i am !!